.gitignore

```diff
@@ -1,12 +1,7 @@
 # Xcode
```

Comment:
This file was copied over from [Relativity](https://github.com/dfed/Relativity/blob/master/.gitignore)